packages/react-scripts/config/webpack.config.ssr.js

```diff
@@ -101,16 +105,16 @@ module.exports = function(webpackEnv) {
     preProcessorOptions = {}
   ) => {
     const loaders = [
-      isEnvDevelopment && require.resolve('style-loader'),
-      isEnvProduction && {
-        loader: MiniCssExtractPlugin.loader,
-        options: Object.assign(
-          {},
-          shouldUseRelativeAssetPaths ? { publicPath: '../../' } : undefined
-        ),
-      },
+      // isEnvDevelopment && require.resolve('style-loader'),
```

Comment:
From what I could see the established style in BRS is to leave in what was already there but comment it out, so that merging in upstream from CRA it's more obvious what you've intentionally removed.